### PR TITLE
Containerize cold extraction module

### PR DESCRIPTION
**Pull request by imiro**
An attempt to #349 for the cold extraction module.

### CODE REVIEW

File: modules/cold-extraction/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  niffler-cold-extraction:
+    build: .
+    volumes:
+    - ./config.json:/usr/src/niffler-cold-extraction/config.json
+    - ./system.json:/usr/src/niffler-cold-extraction/system.json
+    - ./csv/:/usr/src/niffler-cold-extraction/csv/
+    - ./storage:/opt/data/new-study
+    networks:
+      - default
+  orthancc:
```
Comment (pradeeban) [inline modules/cold-extraction/docker-compose.yml]: We don't really use Orthanc in cold-extraction module. Orthanc should be removed from here.

Comment (imiro) [reply modules/cold-extraction/docker-compose.yml]: Noted, I can remove it. It was mentioned in the README and it's useful for testing so I thought to include it.

Comment (pradeeban) [reply modules/cold-extraction/docker-compose.yml]: That is a fair point regarding README. Just it is not really part of the cold-extraction module itself as of now. You can remove it for the sake of consistency.

**Comment by pradeeban** [comment]
Thanks for the fix. Merging since these are new files and will not break anything.

**Pull request merged**